transformato/testing.py

```diff
@@ -0,0 +1,111 @@
+import transformato
+import sys, os
+import logging
+import shutil
+import pathlib
+from transformato import load_config_yaml, SystemStructure, ProposeMutationRoute, IntermediateStateFactory, FreeEnergyCalculator
+import parmed as pm
+import copy
+import numpy as np
+# read in specific topology with parameters
+import subprocess
+
+# list of tuples for values nstep and nstdcd
+from numpy import arange 
+
+def createList(r1,r2,d):
+    return arange(r1,r2+1,d)
+
+
+nsteps_list = createList(10000, 100000, 5000)
+nstdcd_list = createList(1000, 10000, 1000)
+
+steps_map = [(x,y) for x in nsteps_list for y in nstdcd_list]
+
+#user_input = input("Enter the path to destination folder: ")
+#assert os.path.exists(user_input), "There is no folder "+str(user_input)
+
+from transformato import FreeEnergyCalculator
+conf = '/home/master/transformato/config/ethane-methanol-solvation-free-energy.yaml'
```

Comment:
I think this might be better written in more variables/lines, e.g.:
```
name = 'ethane-methanol'
base = '/path/to/config/'
config = f"{base}/{name}-sovlation-free-energy.yaml
```

Reply:
in that way we can also iterate over a list of names